backend/app/Http/Controllers/Auth/RegisterController.php

```
@@ -58,11 +59,38 @@ public function store(RegisterRequest $request): RedirectResponse
+
+        $key = 'resend:' . $request->email;
+        if (RateLimiter::tooManyAttempts($key, 3)) {
+            $seconds = RateLimiter::availableIn($key);
+            return redirect()->route('register')
+                ->with('registered', true)
+                ->with('registered_email', $request->email)
+                ->with('error', "Too many resend attempts. Please wait {$seconds} seconds before trying again.");
+        }
+        RateLimiter::hit($key, 600); // 3 attempts per 10 minutes per email
+
+        $user = User::where('email', $request->email)->first();
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Models\User::where().


(staticMethod.notFound)